service/src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -630,184 +630,6 @@ public boolean resourceExists(UUID workspaceUuid, UUID resourceId) {
     return (count != null && count > 0);
   }
 
-  private void validateUniqueGcsBucket(ControlledGcsBucketResource bucketResource) {
```

Comment:
These methods are all unused, they were replaced by the pattern of putting `UniquenessCheckAttributes` on each resource type